packages/luna/package.json

```
@@ -6,38 +6,83 @@
+    "./components/MultiSelect": "./dist/components/MultiSelect/index.js",
+    "./components/NumericInput": "./dist/components/NumericInput/index.js",
+    "./components/OnlineStatus": "./dist/components/OnlineStatus/index.js",
+    "./components/Popover": "./dist/components/Popover/index.js",
+    "./components/Price": "./dist/components/Price/index.js",
+    "./components/Profile": "./dist/components/Profile/index.js",
+    "./components/Rating": "./dist/components/Rating/index.js",
+    "./components/RawHtml": "./dist/components/RawHtml/index.js",
+    "./components/Select": "./dist/components/Select/index.js",
+    "./components/Settings": "./dist/components/Settings/index.js",
+    "./components/SidebarNav": "./dist/components/SidebarNav/index.js",
+    "./components/Slider": "./dist/components/Slider/index.js",
+    "./components/Stepper": "./dist/components/Stepper/index.js",
+    "./components/Switch": "./dist/components/Switch/index.js",
+    "./components/Textarea": "./dist/components/Textarea/index.js",
+    "./components/TextEditor": "./dist/components/TextEditor/index.js",
+    "./components/Timeline": "./dist/components/Timeline/index.js",
+    "./components/TimePicker": "./dist/components/TimePicker/index.js",
+    "./components/Title": "./dist/components/Title/index.js",
+    "./components/Toast": "./dist/components/Toast/index.js",
+    "./components/utils": "./dist/components/utils/index.js",
+    "./components/VideoPlayer": "./dist/components/VideoPlayer/index.js",
+    "./components/WeekdayPicker": "./dist/components/WeekdayPicker/index.js",
+    "./hooks": "./dist/hooks/index.js",
+    "./hooks/intl": "./dist/hooks/intl.js",
+    "./hooks/validation": "./dist/hooks/validation.js",
+    "./hooks/duration": "./dist/hooks/duration.js",
+    "./hooks/datetime": "./dist/hooks/datetime.js",
+    "./hooks/rule": "./dist/hooks/rule.js",
+    "./hooks/media": "./dist/hooks/media.js",
+    "./locales": "./dist/locales/index.js",
+    "./icons/spinner": "./dist/icons/Spinner.js",
+    "./lib": "./dist/lib/index.js",
+    "./lib/redux": "./dist/lib/redux.js",
+    "./style.css": "./dist/style.css",
+    "./tailwind.config": "./dist/tailwind.config.js",
+    "./tailwind.css": "./dist/tailwind.css",
+    "./svg": "./dist/svg.d.ts"
   },
   "files": [
-    "dist"
+    "dist/**/*",
+    "src/tailwind.css",
+    "tailwind.config.js"
   ],
   "scripts": {
     "dev": "vite",
     "clean": "rm -rf dist/",
-    "build": "tsc && vite build && pnpm build:css",
+    "build": "tsc -p tsconfig.json && tsc-alias -p tsconfig.json && pnpm run build:css",
     "watch:pkg": "vite build -w",
     "watch:build": "concurrently --names \"pkg,css\" \"pnpm watch:pkg\" \"pnpm build:css -w\"",
-    "build:css": "tailwindcss -m -i src/tailwind.css -o dist/style.css",
+    "build:css": "tailwindcss -m -i src/tailwind.css -o dist/style.css && copyfiles --flat src/tailwind.css tailwind.config.js dist/",
```

No need for this, TypeScript will copy the file to dist. 

```suggestion
    "build:css": "tailwindcss -m -i src/tailwind.css -o dist/style.css",
```

---

It didn't do that, even though they were used in `package.json`
```
"./tailwind.config": "./dist/tailwind.config.js",
```